### PR TITLE
Fixes for windows build (audio and math.h)

### PR DESCRIPTION
**Pull request by rgarmsen2295**
Lets windows build again.  There was one change I needed to make outside of cmake in regards to my path variables, so if either of you build on windows at some point let me know...

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -55,14 +55,14 @@ else()
   target_link_libraries(${CMAKE_PROJECT_NAME} ${GLEW_DIR}/lib/libGLEW.a)
 endif()
 
-# FMOD bullshit
+# Get the FMOD environment variable.
 set(FMOD_DIR "$ENV{FMOD_DIR}")
-if (NOT GLEW_DIR)
+if (NOT FMOD_DIR)
```
Comment (joncatanio) [inline CMakeLists.txt:60]: lol thank you.

File: CMakeLists.txt [hunk 2]
```diff
@@ -55,14 +55,14 @@ else()
   target_link_libraries(${CMAKE_PROJECT_NAME} ${GLEW_DIR}/lib/libGLEW.a)
 endif()
 
-# FMOD bullshit
```
Comment (joncatanio) [inline CMakeLists.txt:58]: awwww.

File: CMakeLists.txt [hunk 3]
```diff
@@ -55,14 +55,14 @@ else()
...
+# Get the FMOD environment variable.
 set(FMOD_DIR "$ENV{FMOD_DIR}")
-if (NOT GLEW_DIR)
+if (NOT FMOD_DIR)
    message(FATAL_ERROR "Please point the environment variable FMOD_DIR to the root directory of your FMOD installation.")
 endif()
 include_directories(${FMOD_DIR}/api/lowlevel/inc)
 if(WIN32)
-   target_link_libraries(${CMAKE_PROJECT_NAME} ${FMOD_DIR}/api/lowlevel/lib/libfmod.a)
+   target_link_libraries(${CMAKE_PROJECT_NAME} ${FMOD_DIR}/api/lowlevel/lib/fmod_vc.lib)
```
Comment (joncatanio) [inline CMakeLists.txt:65]: soooo weird

File: src/Shape.h [hunk 4]
```diff
@@ -1,6 +1,8 @@
 #ifndef _SHAPE_H_
 #define _SHAPE_H_
 
+#define _USE_MATH_DEFINES
```
Comment (joncatanio) [inline src/Shape.h:4]: lul

**Review by joncatanio — APPROVED** [review]
You mention there is a change you need to make to build on windows that was external to CMAKE. Put that in the readme so anyone that wants to build the game and play on windows, can. Go ahead and merge after that or whenever you need.

File: CMakeLists.txt (hunk 2 above)
Comment (rgarmsen2295) [reply CMakeLists.txt:58]: Lol, CLEAN code noj, gotta keep it clean xD

**Pull request merged**